lib/__tests__/created_message_queue_handler.test.ts

```diff
@@ -62,16 +63,28 @@ afterEach(() => {
 const aCorrectFiscalCode = "FRLFRC74E04B157I" as FiscalCode;
 const aWrongFiscalCode = "FRLFRC74E04B157" as FiscalCode;
 const anEmail = "x@example.com" as EmailString;
+const aUrl = "http://aUrl.com" as HttpsUrl;
 
 const anEmailNotification: EmailNotification = {
-  channel: {
-    addressSource: NotificationAddressSourceEnum.PROFILE_ADDRESS,
-    // fromAddress: anEmail,
-    toAddress: anEmail
+  channels: {
```

Comment:
that's why it was called "channels" (plural): because we have one notification object with *all* channels metadata for each message : )